src/invert.js

```diff
@@ -12,7 +12,7 @@ var keys = require('./keys');
  * @since v0.9.0
  * @category Object
  * @sig {s: x} -> {x: [ s, ... ]}
- * @param {Object} obj The object or array to invert
+ * @param {Object|*} obj The object or array to invert
```

Comment:
`*` here to me implies that this will take _anything_, including a boolean, a number, etc.  While these won't throw errors, I'm not sure we want to document accidents of implementation this way.